src/exprestive.coffee

```diff
@@ -47,6 +47,23 @@ class Exprestive
       @controllers[camelCase controllerName][controllerAction] arguments...
 
 
+  # Returns if the passed filePath resembles a controller
+  # Controllers are .js or .coffee files that export a constructor and name
+  fileIsController: (filePath) ->
+    # Non-JS files aren't controllers
+    extension = path.extname filePath
+    return false unless extension is '.coffee' or extension is '.js'
+
+    # Controllers must export a name
+    maybeController = require filePath
+    return false unless maybeController.name?
+
+    # Controllers must export a constructor function
+    return false unless typeof maybeController is 'function'
```

Comment:
Do we want to silently skip controllers who's name doesn't end with `Controller`?